### DISCUSSION TITLE
最新版本的tinymce-plugin对应的tinymce版本是多少？

### DISCUSSION THREAD

**Discussion by Minter-Lee** [tinymce-plugin/tinymce-plugin.github.io#10] [Q&A(问题求助)]
亲，还请补充一下你项目当前版本的依赖情况，我刚刚用官方的vue demo 测试你开发的插件，报错如下：

<img width="602" alt="image" src="https://user-images.githubusercontent.com/10607672/198210143-9d763d04-5e85-4ba7-ad80-5d61db2b181f.png">

我怀疑是版本对应问题，我已经更改到了6.0.0 改版本的发布时间已经远早于你最近的提交。还请查明一下，给一个具体依赖。或者发布新版本来fix 这个问题

**Comment by Five-great** [comment]
6版本 需要 ***@***.***
可以加QQ群或者我QQ 985540932&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tinymce-plugin/tinymce-plugin.github.io"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月27日(星期四) 下午2:43
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[tinymce-plugin/tinymce-plugin.github.io] 最新版本的tinymce-plugin对应的tinymce版本是多少？ (Discussion #10)





 
亲，还请补充一下你项目当前版本的依赖情况，我刚刚用官方的vue demo 测试你开发的插件，报错如下：
 

 
我怀疑是版本对应问题，我已经更改到了6.0.0 改版本的发布时间已经远早于你最近的提交。还请查明一下，给一个具体依赖。或者发布新版本来fix 这个问题
